tests/e2e/test_rosacli_cluster.go

```diff
@@ -1645,6 +1646,146 @@ var _ = Describe("HCP cluster creation negative testing",
 						ContainSubstring(
 							"ERR: Additional Allowed Principals is supported only for Hosted Control Planes"))
 			})
+
+		It("HCP cluster creation subnets validation - [id:72538]",
+			labels.High, labels.Runtime.Day1Negative,
```

Comment:
Day1 negative cases has no profile setting.

Reply:
https://github.com/openshift/rosa/blob/master/tests/e2e/test_rosacli_cluster.go#L1431

Reply:
should be fine then